cherrypy/test/test_states.py

```diff
@@ -125,6 +125,9 @@ def exittest():
         engine.block()
         self.assertEqual(engine.state, engine.states.EXITING)
 
+    @pytest.mark.skipif(
```

Comment:
Tests mustdn't be skipped just because they fail. Instead, xfail should be used: https://pganssle-talks.github.io/xfail-lightning/

Reply:
They don’t fail.  They hang so xfail would be a useless change.

Reply:
@cclauss `skipif` is still semantically wrong here. `xfail` has `run=False` exactly for this case.

P.S. https://github.com/cherrypy/cheroot/pull/331 that I'm planning to merge soon should fix this problem.

Reply:
P.P.S. Also, the best way to isolate this test is to use `pytest-forked` so that one test wouldn't crash the whole process.

Reply:
Removed because 331 should fix this.